src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -654,7 +659,7 @@ public RepairUnit addRepairUnit(RepairUnit.Builder newRepairUnit) {
           && repairUnitRow.getSet("column_families", String.class).equals(params.columnFamilies)
           && repairUnitRow.getBool("incremental_repair") == params.incrementalRepair
           && repairUnitRow.getSet("nodes", String.class).equals(params.nodes)
-          && repairUnitRow.getSet("datacenters", String.class).equals(params.datacenters)
+          && repairUnitRow.getSet("\"datacenters\"", String.class).equals(params.datacenters)
```

Comment:
Is this really necessary ? I would expect the name to be `datacenters` and not `"datacenters"`